pkg/controller/directvolumemigration/rsync.go

```diff
@@ -155,17 +155,17 @@ func (t *Task) areRsyncTransferPodsRunning() (bool, error) {
 							"pod", path.Join(pod.Namespace, pod.Name),
 							"podPhase", pod.Status.Phase,
 							"podConditionMessage", podCond.Message)
-						return false, nil
+						return false, &pod, nil
```

Comment:
This is a good improvement. I think we should collect _all_ non-running Rsync Pods instead of returning on the first one. What do you think?

Reply:
+1, I will adapt the PR.

Reply:
@pranavgaikwad were you thinking we would also provide all of the commands to check on individual Pods, or just one?

Reply:
@djwhatle the typical pattern I have followed elsewhere (PV resizing/Retry) is to raise a warning with names of problematic resources. I will defer to you on this one. Maybe one command with placeholders and names of the resources followed by that, or perhaps, just text explaining how to use debug view to get to the root of the problem should be fine I think.

Reply:
Now that I am thinking about it, I think redirecting users to the debug view should be the default go-to pattern for such warnings. Open to discuss pros/cons of that, but seems like a good way to get users used to the debug tree.

Reply:
> Now that I am thinking about it, I think redirecting users to the debug view should be the default go-to pattern for such warnings. Open to discuss pros/cons of that, but seems like a good way to get users used to the debug tree.

I feel this pointer should exist in the UI, but not on the API messaging. It would be weird for a user who is doing a purely CLI driven migration to get directed to the UI, and might not even be an option for them. We can provide this indicator anytime a warning shows up during a migration (see debug tree for more info).